openmc/mgxs/mgxs.py

```diff
@@ -69,6 +69,21 @@ def _df_column_convert_to_bin(df, current_name, new_name, values_to_bin,
                               reverse_order=False):
     """Convert a Pandas DataFrame column from the bin edges to an index for
     each bin. This method operates on the DataFrame, df, in-place.
+
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        A Pandas DataFrame containing the cross section data.
+    current_name : str
+        Name of the column to replace with bins
+    new_name : str
+        New name for column after the data is replaced with bins
+    values_to_bin : Iterable of Real
+        Values of the bin edges to be used for identifying the bins
+    reverse_order : bool
+        Whether or not the bin indices should be reversed like for energy group
```

Comment:
Oh I see what you mean since this method is general. How about simply "Whether the bin indices should be reversed"